cmake/projects/Catch/hunter.cmake

```diff
@@ -39,7 +39,7 @@ hunter_add_version(
     URL
     "https://github.com/hunter-packages/Catch/archive/v1.5.9-hunter.tar.gz"
     SHA1
-    737cb1c98fedccceb95e7bfd385e5dea0ad5d047
```

Comment:
Do you have old archive with this hash? I can upload old files so it will not affect caches and dependencies. Take a look at `~/.hunter/Download` directory: https://docs.hunter.sh/en/latest/reference/layouts/deployed.html#download

Reply:
yes, uploaded to https://github.com/dvirtz/uploads/blob/master/v1.5.9-hunter.tar.gz

Reply:
It's not an archive, it's some HTML page. I think you've used wrong link.

Reply:
sorry, try https://github.com/dvirtz/uploads/raw/master/v1.5.9-hunter.tar.gz

Reply:
Released: https://github.com/ruslo/hunter/releases/tag/v0.19.176